std_msgs/Header header
uint32 sending_agent
Person[] persons